frontend/src/metabase/core/components/Icon/icons/index.ts

```diff
@@ -335,6 +335,8 @@ import variable_component from "./variable.svg?component";
 import variable_source from "./variable.svg?source";
 import verified_component from "./verified.svg?component";
 import verified_source from "./verified.svg?source";
+import verified_filled_component from "./verified_filled.svg?component";
```

Comment:
The icon not centered vertically nor horizontally.
 
![image](https://github.com/metabase/metabase/assets/6830683/33a84bc2-55c9-4540-a061-1f45253aec3c)